InvenTree/order/models.py

```diff
@@ -612,19 +620,14 @@ def filterByDate(queryset, min_date, max_date):
 
         return queryset
 
-    def save(self, *args, **kwargs):
-
-        self.rebuild_reference_field()
```

Comment:
Question: Isn't this needed to keep the sorting going?

Reply:
It's already performed in the save() method for the parent `Order` class. This is just deleting extra code. I'll ensure there is a unit test for this though.